solutions/swb-reference/integration-tests/config/example.yaml

```diff
@@ -21,6 +21,9 @@
 #researcher1UserNameParamStorePath: ''
 #researcher1PasswordParamStorePath: ''
 
+## Default external ID
+#externalId: 'workbench'
```

Comment:
Can we have this stored in an SSM parameter instead? This will prevent us storing the value publicly

Reply:
for the scope of the ticket AC, storing in SSM is considering the next step. I can create a follow up ticket for this

Reply:
For now, can we remove 'workbench'? Leave this empty